user_guide/src/timeseries/parsing_dates_times.md

```diff
@@ -0,0 +1,39 @@
+# Parsing dates and times
+
+## Datatypes
+
+`Polars` inherits the following datetime datatypes from Apache Arrow:
```

Comment:
We don't use Arrow naming convention, so I would like to phrase this as "Polars has the folling datatime datatypes". 

Which arrow types we use is an implementation detail.